### DISCUSSION TITLE
Enabling only one working file during test.

### DISCUSSION THREAD

**Discussion by nkadoic** [SafeExamBrowser/seb-win-refactoring#568] [Q&A]
Dear people, 
I am interested in using SEB for BYOD tests in Moodle. My tests are implemented in Moodle test and Microsoft Excel. So, the tests are generated in Moodle, and students must calculate in Excel. Then they input the solutions in Moodle tests (for automatic grading) and upload the excel file they created. 
I saw that Excel could be enabled during the tests in SEB. However, this is not enough for me because students open different Excel files from their computers and can see how something is solved in those files. What I need is Excel enabled for only one file. I can prepare this file with assignments inserted in it. I want to disable that they open other Excel files or work in one mega file (they merge all Excel files from teaching hours in one Excel file; they use the sheets when needed, and delete what they don't need before the test end)?
Any idea on this - is it possible?

**Comment by dbuechel** [comment]
In short: No, especially for BYOD. On managed systems, you could prepare the computers the way you'd like to have them, but on BYOD there's really no way to do this. This would require that there is a functionality in Excel where you could restrict which documents may or may not be opened, and this is (to my knowledge) not possible.
